cmd/get.go

```diff
@@ -77,5 +113,11 @@ func init() {
 		"Prints a specific version from the changelog to stdout.",
 	)
 
+	getCmd.Flags().Var(
+		&outputTemplate,
+		"output",
+		fmt.Sprintf(`Output template. allowed: "%s" or "%s"`, outputStandard, outputNotes),
+	)
```

Comment:
For the flag definition, we can use `StringVar` and set a default value of `standard`.

```suggestion
	getCmd.Flags().StringVar(
		&outputType,
		"output",
		"standard",
		fmt.Sprintf(`Output template. allowed: "%s" or "%s"`, "standard", "notes"),
	)
```

Reply:
Where does flag value validation happen?